packages/webapp/src/pages/delegates/index.tsx

```diff
@@ -184,8 +170,8 @@ const Chiefs = () => {
             </Container>
             {chiefsAsMembers.map((delegate) => {
                 if (
-                    !delegate ||
-                    delegate.account === electionState.lead_representative
+                    !delegate // ||
+                    // delegate.account === electionState.lead_representative
                 )
                     return null;
                 return (
```

Comment:
For some reason, the `-mt-px` in the `<div>` below this in line 192 is now _creating_ the double-wide border. Not sure what changed. If that's happening on your end to, you can just remove that class name. 🤷‍♂️

Reply:
![image](https://user-images.githubusercontent.com/7911424/128435156-5edab83c-05c7-46dc-8aea-acb2b80792e4.png)